systemtest/src/test/resources/connect-build/build-connect-image.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+RANDOM=$(shuf -i 60000-99999 -n 1)
+
+KAFKA_VERSION=$1
+BASE_IMAGE=$2
+CONNECT_IMAGE=${3:-"strimzi/connect:$RANDOM"}
+
+function getLatestKafkaVersionFromYAML() {
+  local strimziRoot=$(git rev-parse --show-toplevel)
+  KAFKA_VERSION=$(cat $strimziRoot/kafka-versions.yaml | yq eval '.[] | select(.default) | .version' -)
```

Comment:
In case the `kafka-versions.yaml` is not in the root folder the script should simply fail -> in case user will not specify any Kafka version and we cannot get it from the YAML, it should not continue -> because we need it for downloading the `.jar` of file-sink.
Or what error handling do you mean? Some check that the `kafka-versions.yaml` exists and if not, `exit 1` and error message? (The error message cannot be there using echo or printf, as we should ideally use `eval $(.script)`, which would fail because "unknown command)

Reply:
I was thinking about something like this:
```bash
function getLatestKafkaVersionFromYAML() {
  local strimziRoot=$(git rev-parse --show-toplevel)
  local kafkaVersionsFile="$strimziRoot/kafka-versions.yaml"
  
  # Check if kafka-versions.yaml exists
  if [[ ! -f $kafkaVersionsFile ]]; then
    echo "[ERROR] File $kafkaVersionsFile does not exist."
    exit 1
  fi
  
  # Retrieve the Kafka version
  KAFKA_VERSION=$(cat $kafkaVersionsFile | yq eval '.[] | select(.default) | .version' -)
  
  # Check if the version is retrieved successfully
  if [[ -z $KAFKA_VERSION ]]; then
    echo "[ERROR] Failed to retrieve Kafka version from $kafkaVersionsFile."
    exit 1
  fi
}
```
But when you mention `eval`...I know that current implementation is okay ;)